FitOn/templates/profile.html

```diff
@@ -24,11 +24,122 @@
                     </div>
                 </div>             
 
+<<<<<<< HEAD
```

Comment:
Some merge conflicts issues are not properly resolved before pushing

Reply:
resolved